test/runner/requirements/constraints.txt

```diff
@@ -17,3 +17,4 @@ ntlm-auth >= 1.0.6 # message encryption support
 requests-ntlm >= 1.1.0 # message encryption support
 requests-credssp >= 0.1.0 # message encryption support
 voluptuous >= 0.11.0 # Schema recursion via Self
+botocore >= 1.9.23 # latest aws services
```

Comment:
Could you include a more specific comment about why this version is required, such as what feature(s) were added, or bug(s) were fixed? Please use the lowest version required.

Reply:
Updated.   This version of botocore introduces secretsmanager, fms, and acm-pca as valid services.